pages/news.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col>
+        <h1>{{ $t('news.title') }}</h1>
+        <p>{{ $t('news.blurb') }}</p>
+        <v-card v-for="(event, i) in newsCollection" :key="i">
+          <v-card-title class="info">
+            {{ event.title }}
+          </v-card-title>
+          <v-card-subtitle class="info">
+            <span class="blue--text text--darken-3">{{ event.date }}</span>
+            <span class="ml-2">{{ event.keywords.join(',') }}</span>
+          </v-card-subtitle>
+          <v-card-text class="pt-3" v-html="event.content" />
```

Comment:
pages\news.vue
  15:37  warning  'v-html' directive can lead to XSS attack  vue/no-v-html

✖ 1 problem (0 errors, 1 warning)

Reply:
Don't know if there's any way around this to make it work the way the current news page works, allowing HTML elements into the bodies of news items.
All the news items will come from our own internal databases so (I certainly hope) there will be no attacks through that.

Reply:
It's actually the variable `event.content` that can be altered through an XSS attack. Not necessarily the news content from our DB getting attacked. Having the news come back as a JSON would make the most sense, just like in WDR.

So thus you can use `v-text="news.content"`; or something like:
```
<v-card>
  <v-card-title>{{news.title}}</v-card-title>
  <v-card-subtitle>{{news.date}}</v-card-subtitle>
  <v-card-text>{{news.content}}</v-card-text>
</v-card>
```

Reply:
Oh I also wanted to add: let's avoid using the variable name `event` here. Usually when we use `event` in a variable name, it references a JS event handler, rather than the "event" of a news collection.